### PR TITLE
docs: add project roadmap

### PR DESCRIPTION
**Pull request by pi0**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Add project roadmap to the community section and link from landing.

Preview: https://deploy-preview-3448--nuxt3-docs.netlify.app/community/roadmap

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/4.community/4.roadmap.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Release             | Expected date | Description
+--------------------|---------------|---------------------------------------------------
+`nuxt@3.0.0-rc.1`   | March 2022    | Nuxt v3 first release candidate
+`nuxt@2.16`         | March 2022    | Nuxt v2 cumulative updates for future compatibility
+`nuxt@3.0.0`        | June  2022    | Nuxt v3 stable release
+
+### Current Releases
+
+Currently, [Nuxt 2](https://github.com/nuxt/nuxt.js) is considered **stable** version.
+Lastest updates on Nuxt 2.x are available via `nuxt-edge` npm package and expected to release alongside first release candidate of Nuxt 3 for future compatibility. [Nuxt 3](https://v3.nuxtjs.org) is **under development** and available as `nuxt3` npm package with automated releases for beta testing.
```
Comment (danielroe) [inline docs/content/4.community/4.roadmap.md:19]:
```suggestion
Currently, [Nuxt 2](https://github.com/nuxt/nuxt.js) is considered the **stable** version.

The latest updates on Nuxt 2.x are available via the `nuxt-edge` npm package and are expected to be released alongside the first release candidate of Nuxt 3 for future compatibility. [Nuxt 3](https://v3.nuxtjs.org) is **under development** and available as the `nuxt3` npm package with automated releases for beta testing.
```

File: docs/content/4.community/4.roadmap.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+---------|----|---------|--------------|------|-----------------
+Nuxt 2.x | [`nuxt`](https://npmjs.com/package/nuxt)  | Maintenance | <a href="https://npmjs.com/package/nuxt"><img src="https://flat.badgen.net/npm/v/nuxt"></a>   | [2.x docs](https://nuxtjs.org/docs) | [nuxt/nuxt.js#2.x](https://github.com/nuxt/nuxt.js/tree/2.x)
+Nuxt 2.x (edge*) | [`nuxt-edge`](https://npmjs.com/package/nuxt) | Maintenance | <a href="https://npmjs.com/package/nuxt-edge"><img src="https://flat.badgen.net/npm/v/nuxt-edge"></a>      | [2.x docs](https://nuxtjs.org/docs) | [nuxt/nuxt.js](https://github.com/nuxt/nuxt.js)
+Nuxt 3.x (edge*) | [`nuxt3`](https://npmjs.com/package/nuxt3) | Development        | <a href="https://npmjs.com/package/nuxt3"><img src="https://flat.badgen.net/npm/v/nuxt3"></a>       | [3.x docs](https://v3.nuxtjs.org/) | [nuxt/framework](https://github.com/nuxt/framework)
+
+(*) Edge releases are automated after each commit passing tests
+
+## 🛣️ Roadmap
+
+In roadmap below, we listed major expected features that comming soon with Nuxt 3.
```
Comment (danielroe) [inline docs/content/4.community/4.roadmap.md:31]:
```suggestion
In roadmap below are the major expected features that are coming soon with Nuxt 3.
```

File: docs/content/4.community/4.roadmap.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Nuxt 3 RC | Q1 2022 | [nuxt/framework#3447](https://github.com/nuxt/framework/discussions/3447) | Nuxt v3 first release candidate
+Extending | Q1 2022 | [nuxt/framework#3222](https://github.com/nuxt/framework/issues/3222) | Native extending and theme support
+Content v2 | Q1 2022     | -      | A rewrite of [nuxt/content](https://github.com/nuxt/content) for Nuxt 3 support
+Devtools | Q2 2022       | -      | An integrated devtools experience for Nuxt
+Auth     | Q2 2022       | -      | A rewrite of [nuxt-community/auth-module](https://github.com/nuxt-community/auth-module) for Nuxt 3 support
+Seo & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and Server-worker support
+Translations | Q2/Q3 2022 | [nuxt/translations#4](https://github.com/nuxt/translations/discussions/4) (private) | A collabrative project for stable translation process of Nuxt 3 docs. Currently pending for ideas and documentation tooling support (content v2 with remote sources).
```
Comment (danielroe) [inline docs/content/4.community/4.roadmap.md:46]:
```suggestion
Nitropack  | Q1 2022 | [nuxt/framework#3161](https://github.com/nuxt/framework/issues/3161) | Stable and full-featured server engine for Nuxt 3
Generate and Caching | Q1 2022 | [nuxt/framework#3161](https://github.com/nuxt/framework/issues/3161) | Route caching rules and full static generation
Test Utils   | Q1 2022 | [nuxt/framework#3198](https://github.com/nuxt/framework/issues/3198) | A rewrite of [nuxt/test-utils](https://github.com/nuxt/test-utils) for testing Nuxt 3 and new modules
Nuxt 3 RC | Q1 2022 | [nuxt/framework#3447](https://github.com/nuxt/framework/discussions/3447) | Nuxt v3 first release candidate
Extending | Q1 2022 | [nuxt/framework#3222](https://github.com/nuxt/framework/issues/3222) | Native extending and theme support
Content v2 | Q1 2022     | -      | A rewrite of [nuxt/content](https://github.com/nuxt/content) for Nuxt 3 support
Devtools | Q2 2022       | -      | An integrated devtools experience for Nuxt
Auth     | Q2 2022       | -      | A rewrite of [nuxt-community/auth-module](https://github.com/nuxt-community/auth-module) for Nuxt 3 support
Seo & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and Server-worker support
Translations | Q2/Q3 2022 | [nuxt/translations#4](https://github.com/nuxt/translations/discussions/4) (private) | A collaborative project for a stable translation process for Nuxt 3 docs. Currently pending for ideas and documentation tooling support (content v2 with remote sources).
```

File: docs/content/4.community/4.roadmap.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+## 📦 Core Modules
+
+Alongside with Nuxt Framework, there are modules that are vital for ecosystem. Their status will be updated below.
```
Comment (danielroe) [inline docs/content/4.community/4.roadmap.md:50]:
```suggestion
In addition to the Nuxt framework, there are modules that are vital for the ecosystem. Their status will be updated below.
```

**Review by danielroe — APPROVED** [review]

File: docs/content/4.community/4.roadmap.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+Test Utils   | Q1 2022 | [nuxt/framework#3198](https://github.com/nuxt/framework/issues/3198) | A rewrite of [nuxt/test-utils](https://github.com/nuxt/test-utils) for testing Nuxt 3 and new Modules
+Nuxt 3 RC | Q1 2022 | [nuxt/framework#3447](https://github.com/nuxt/framework/discussions/3447) | Nuxt v3 first release candidate
+Extending | Q1 2022 | [nuxt/framework#3222](https://github.com/nuxt/framework/issues/3222) | Native extending and theme support
+Content v2 | Q1 2022     | -      | A rewrite of [nuxt/content](https://github.com/nuxt/content) for Nuxt 3 support
+Devtools | Q2 2022       | -      | An integrated devtools experience for Nuxt
+Auth     | Q2 2022       | -      | A rewrite of [nuxt-community/auth-module](https://github.com/nuxt-community/auth-module) for Nuxt 3 support
+Seo & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and Server-worker support
```
Comment (manniL) [inline docs/content/4.community/4.roadmap.md:45]:
```suggestion
SEO & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and service worker support
```

File: docs/content/4.community/4.roadmap.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+# Project Roadmap
```
Comment (Atinux) [inline docs/content/4.community/4.roadmap.md:1]:
```suggestion
# Roadmap
```

This is explicit enough :)

File: docs/content/1.getting-started/1.introduction.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ Getting started with Nuxt 3 is straightforward.
 
 ::alert{type=warning icon=🚧}
 Nuxt 3 is currently in beta, keep in mind that **it is not yet production-ready**.<br>
-Thank you in advance for your understanding 💛
+Thank you in advance for your understanding 💛 Check out [Project Roadmap](/community/roadmap) for more info.
```
Comment (Atinux) [inline docs/content/1.getting-started/1.introduction.md:7]:
```suggestion
Thank you in advance for your understanding 💛 Check out the [roadmap](/community/roadmap) for more info.
```

File: docs/content/4.community/4.roadmap.md (hunk 3 above)
Comment (Atinux) [reply docs/content/4.community/4.roadmap.md]: Seems I cannot commit this change @danielroe

File: docs/content/4.community/4.roadmap.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+SEO & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and service worker support
+Translations | Q2/Q3 2022 | [nuxt/translations#4](https://github.com/nuxt/translations/discussions/4) (private) | A collabrative project for stable translation process of Nuxt 3 docs. Currently pending for ideas and documentation tooling support (content v2 with remote sources).
+
+## 📦 Core Modules
+
+In addition to the Nuxt framework, there are modules that are vital for the ecosystem. Their status will be updated below.
+
+Module         | Status              | Nuxt Support | Repository | Description
+---------------|---------------------|--------------|------------|-------------------
+Content 1.x    | Maintenance         | 2.x          | [nuxt/content](https://github.com/nuxt/content) | EOL
```
Comment (manniL) [inline docs/content/4.community/4.roadmap.md:55]: Question: Why is 1.x marked as "EOL" when there is not "alternative" yet? 🤔

File: docs/content/4.community/4.roadmap.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+Nuxt 3 RC | Q1 2022 | [nuxt/framework#3447](https://github.com/nuxt/framework/discussions/3447) | Nuxt v3 first release candidate
+Extending | Q1 2022 | [nuxt/framework#3222](https://github.com/nuxt/framework/issues/3222) | Native extending and theme support
+Content v2 | Q1 2022     | -      | A rewrite of [nuxt/content](https://github.com/nuxt/content) for Nuxt 3 support
+Devtools | Q2 2022       | -      | An integrated devtools experience for Nuxt
+Auth     | Q2 2022       | -      | A rewrite of [nuxt-community/auth-module](https://github.com/nuxt-community/auth-module) for Nuxt 3 support
+SEO & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and service worker support
+Translations | Q2/Q3 2022 | [nuxt/translations#4](https://github.com/nuxt/translations/discussions/4) (private) | A collabrative project for stable translation process of Nuxt 3 docs. Currently pending for ideas and documentation tooling support (content v2 with remote sources).
```
Comment (danielroe) [inline docs/content/4.community/4.roadmap.md:47]:
```suggestion
Nitropack  | Q1 2022 | [nuxt/framework#3161](https://github.com/nuxt/framework/issues/3161) | Stable and full-featured server engine for Nuxt 3
Generate and Caching | Q1 2022 | [nuxt/framework#3161](https://github.com/nuxt/framework/issues/3161) | Route caching rules and full static generation
Test Utils   | Q1 2022 | [nuxt/framework#3198](https://github.com/nuxt/framework/issues/3198) | A rewrite of [nuxt/test-utils](https://github.com/nuxt/test-utils) for testing Nuxt 3 and new modules
Nuxt 3 RC | Q1 2022 | [nuxt/framework#3447](https://github.com/nuxt/framework/discussions/3447) | Nuxt v3 first release candidate
Extending | Q1 2022 | [nuxt/framework#3222](https://github.com/nuxt/framework/issues/3222) | Native extending and theme support
Content v2 | Q1 2022     | -      | A rewrite of [nuxt/content](https://github.com/nuxt/content) for Nuxt 3 support
Devtools | Q2 2022       | -      | An integrated devtools experience for Nuxt
Auth     | Q2 2022       | -      | A rewrite of [nuxt-community/auth-module](https://github.com/nuxt-community/auth-module) for Nuxt 3 support
SEO & PWA     | Q2 2022       | [nuxt/framework#1823](https://github.com/nuxt/framework/discussions/1823) | Migrating from [nuxt-community/pwa-module](https://github.com/nuxt-community/auth-module) for built-in SEO utils and service worker support
Translations | Q2/Q3 2022 | [nuxt/translations#4](https://github.com/nuxt/translations/discussions/4) (private) | A collaborative project for a stable translation process for Nuxt 3 docs. Currently pending for ideas and documentation tooling support (content v2 with remote sources).
```

File: docs/components/molecules/HomeHero.vue [hunk 10]
```diff
@@ -3,9 +3,9 @@
     <div class="flex flex-wrap justify-center py-0 section d-container-content">
       <section class="flex flex-col justify-start w-full px-4 pb-24 sm:pt-12 sm:pb-36 lg:pt-24 lg:pb-50 text-center z-20">
         <div class="highlight-box mx-auto mt-6 rounded-md">
-          <a href="https://nuxtjs.org/announcements/nuxt3-beta" target="_blank" rel="noopener" class="text-sm inline-flex items-center space-x-1 px-4 md:px-5 py-1 md:py-2 font-medium">
-            <span>Read the announcement</span><IconArrowRight class="mt-1px" />
-          </a>
+          <NuxtLink to="/community/roadmap" class="text-sm inline-flex items-center space-x-1 px-4 md:px-5 py-1 md:py-2 font-medium">
+            <span>Read Nuxt 3 roadmap</span><IconArrowRight class="mt-1px" />
```
Comment (danielroe) [inline docs/components/molecules/HomeHero.vue:7]:
```suggestion
            <span>Check out the Nuxt 3 roadmap</span><IconArrowRight class="mt-1px" />
```

File: docs/content/4.community/4.roadmap.md (hunk 8 above)
Comment (pi0) [reply docs/content/4.community/4.roadmap.md:55]:
```suggestion
Content 1.x    | Maintenance         | 2.x          | [nuxt/content](https://github.com/nuxt/content) | maintenance only
```

**Review by Atinux — APPROVED** [review]

**Pull request merged**